src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -2601,6 +2601,23 @@ def test_one_deploy(self, resource_group):
             JMESPathCheck('complete', True)
         ])
 
+    @live_only()
```

Comment:
May I ask why this test is marked as `@live_only()`?

Reply:
The original OneDeploy test was marked as `@live_only()` so I followed the same pattern. I can remove it if preferred.